core/schema/module.go

```diff
@@ -993,6 +993,9 @@ func (s *moduleSchema) updateCodegenAndRuntime(
 				// already has some config for the file
 				continue
 			}
+			if fileName[0] == '/' {
+				fileName = fileName[1:]
+			}
```

Comment:
`strings.TrimPrefix("/")`

SGTM though, don't think this needs tests.

Reply:
How do I forgot the trim prefix function. I will do fix it in 1-2 hours.

Reply:
Fixed. And re-run all failed jobs.